docs/content/preview/yugabyte-cloud/cloud-monitor/cloud-advisor.md

```diff
@@ -0,0 +1,132 @@
+
+Indexes can slow down some operations. For example:
+
+- When doing an INSERT or DELETE on a table, indexes need to be updated to remain consistent with the data in the main table. An index can also prevent a basic insert or delete from being executed as a single row transaction.
+
+- When doing an UPDATE on a table, if the modified column is part of the index, then the index must be updated as well.
+
+**Fix the problem**
+
+Connect to the database and use DROP INDEX to delete the unused indexes.
+
+### Schema suggestions
+
+Advisor scans for indexes that can benefit from using range sharding instead of the default hash sharding.
+
+**Why**
+
+Range sharding is more efficient for queries that look up a range of rows based on column values that are less than, greater than, or that lie between some user specified values. For example, with timestamp columns, most queries use the timestamp column to filter data from a range of timestamps.
+
+The following example shows a query that filters the data using a specific time window. First, create an `order_details` table:
+
+```sql
+CREATE TABLE order_details (
+   order_id smallint NOT NULL,
+   product_id smallint NOT NULL,
+   unit_price real NOT NULL,
+   order_updated  timestamp NOT NULL,
+   PRIMARY KEY (order_id)
+);
+```
+
+Then create an index on `order_updated` using range sharding:
+
+```sql
+CREATE INDEX ON order_details (order_updated asc);
+```
+
+The following query finds the number of orders in a specific time window:
+
+```sql
+SELECT count(*) FROM order_details
+  WHERE order_updated > '2018-05-01T15:14:10.386257'
+  AND order_updated < '2018-05-01T15:16:10.386257';
+```
+
+**Fix the problem**
+
+Connect to the database and use DROP INDEX to delete the indexes, and then recreate the indexes using range sharding. For more information on sharding strategies, refer to [Sharding data across nodes](../../../explore/linear-scalability/sharding-data/).
+
+### Connection skew
+
+Advisor scans node connections to determine whether some nodes are loaded with more connections than others. Advisor flags any node handling 50% more connections than the other nodes in the past hour.
+
+**Why**
+
+Connections should be distributed equally across all the nodes in the cluster. Unequal distribution of connections can result in hot nodes, causing higher resource use on those nodes. Excessive workload on a specific node can potentially cause stability issues.
+
+**Fix the problem**
+
+- If a load balancer is used to distribute connections among nodes of cluster, review the configuration of the load balancer. If you are using the YugabyteDB Managed load balancer, contact {{<support-cloud>}}.
+- If you are load balancing in your application or client, review your implementation.
+
+### Query load skew
+
+Advisor scans queries to determine whether some nodes are handling more queries than others. Advisor flags nodes that processed 50% more queries than the other nodes in the past hour.
```

Comment:
We changed the implementation a bit. 

Advisor flags nodes that processed 50% more queries than the other nodes in the past hour or when the perf advisor scan was performed  last time.